sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyWrapResult.java

```diff
@@ -0,0 +1,48 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+/**
+ * Represents the details of wrap operation result.
+ */
+public class KeyWrapResult {
+
+    public KeyWrapResult(byte[] encryptedKey, KeyWrapAlgorithm algorithm) {
+        this.encryptedKey = encryptedKey;
+        this.algorithm = algorithm;
+    }
+
+    /**
+     * The encrypted key content
+     */
+    private byte[] encryptedKey;
+
+    /**
+     * The key wrap algorithm used to wrap the key content.
+     */
+    private KeyWrapAlgorithm algorithm;
+
+    /**
+     * Get the encrypted key content.
+     * @return The encrypted key.
+     */
+    public byte[] encryptedKey() {
+        return encryptedKey;
```

Comment:
Return this as a clone?

Reply:
This might need to be consistent across languages, will ask the KV team for P3 and see what they say.